pom.xml

```diff
@@ -296,6 +324,11 @@
                                     <include>org.checkerframework:checker-qual</include>
                                     <include>org.checkerframework:checker-compat-qual</include>
                                     <include>org.codehaus.mojo:animal-sniffer-annotations</include>
+                                    <include>com.fasterxml.jackson.core:*</include>
+                                    <include>com.fasterxml.jackson.dataformat:*</include>
+                                    <include>com.fasterxml.jackson.module:*</include>
+                                    <!-- Include paranamer required by jackson-module-scala -->
```

Comment:
Comment not needed as we do not document the reason for other transient dependencies above.

`mvn dependency:tree` shows you quickly where dependencies come from, and we want to include them all here.